### DISCUSSION TITLE
Endless Runner

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#142] [Games]
# Endless Runner

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/endless-runner/?giscus=c8b47b84546f246f7e55f1d9Ykrd4DS4fZPjdsXeMPtfiebmgM2mAIugtiHGGjVmLVUMYn4S6vlSb1VgHrNcOZI4gryatZId6P3H8ibB9p6s%2BY9xxF%2FfyY%2BvvkufoenyM5j8vctj46XtfJ1%2BHSs%3D

**Comment by DenialAdams** [comment]
I programmed this game using Roland, my own work-in-progress programming language. It features wasm4 as a native compilation target :)

You can find the source code for this game here: https://github.com/DenialAdams/InfiniteRunner/blob/master/infinite-runner.rol

More about Roland can be found here: https://github.com/DenialAdams/roland
